index.html

```diff
@@ -181,17 +187,25 @@ <h5>
             </div>
             <div class="row py-xl-5">
                 <div class="col-12">
-                    <p><img alt="" src="images/icon-phone.svg"/> Phone: +1-543-123-4567</p>
+                    <a href="tel:1.543.123.4567">
+                        <p><img alt="" src="images/icon-phone.svg"/> Phone: +1-543-123-4567</p>
+                    </a>
                 </div>
                 <div class="col-12">
-                    <p><img alt="" src="images/icon-email.svg"/> example@complany.com</p>
+                    <a href="mailto:example@complany.com">
+                        <p><img alt="" src="images/icon-email.svg"/> example@complany.com</p>
+                    </a>
                 </div>
                 <div class="col-10 col-md-6">
```

Comment:
- col-md-7
- remove the height on these `fab`'s

Reply:
Fixed